doc/user_guide/encoding.rst

```diff
@@ -50,16 +50,16 @@ x           :class:`X`           The x-axis value                   :ref:`galler
 y           :class:`Y`           The y-axis value                   :ref:`gallery_scatter_tooltips`
 x2          :class:`X2`          Second x value for ranges          :ref:`gallery_gantt_chart`
 y2          :class:`Y2`          Second y value for ranges          :ref:`gallery_candlestick_chart`
-longitude   :class:`Longitude`   Longitude for geo charts           :ref:`gallery_airports`
-latitude    :class:`Latitude`    Latitude for geo charts            :ref:`gallery_airports`
+longitude   :class:`Longitude`   Longitude for geo charts           :ref:`gallery_airport_connections`
+latitude    :class:`Latitude`    Latitude for geo charts            :ref:`gallery_airport_connections`
```

Comment:
Should be changed to `gallery_point_map`. This was previous the `gallery_airports` example.

Reply:
Thanks! Fixed in my latest commit